clients/lsp-javascript.el

```diff
@@ -44,7 +44,7 @@
 
 (defun lsp-typescript-javascript-tsx-jsx-activate-p (filename &optional _)
   "Check if the javascript-typescript language server should be enabled based on FILENAME."
-  (or (string-match-p (rx (one-or-more anything) "." (or "ts" "js") (opt "x") string-end) filename)
+  (or (string-match-p (rx (one-or-more anything) "." (or "mjs" (or "ts" "js") (opt "x")) string-end) filename)
```

Comment:
"mjs" should be part of the `or`, after "js", something like that (beware indentation!):
```suggestion
  (or (string-match-p (rx (one-or-more anything) "." (or "ts" "js" "mjs") (opt "x") string-end) filename)
 ```

Reply:
Nevermind.